tests/autostart_test.py

```diff
@@ -136,10 +136,8 @@ async def test_autostart(client: AsyncClient, jupyter: MockJupyter) -> None:
                 "business": {
                     "failure_count": 0,
                     "image": {
-                        "description": "Recommended (Weekly 2077_43)",
-                        "reference": (
-                            "lighthouse.ceres/library/sketchbook:recommended"
-                        ),
+                        "description": ANY,
+                        "reference": ANY,
```

Comment:
Why do we lose the ability to test specific descriptions and references here?

Reply:
(and elsewhere in these tests)

Reply:
Because it wasn't clear to me that we would want to set them directly or use the ones from the NubladoClient testing suite, and there they are not exposed well.  I think.